tests/rptest/services/admin_ops_fuzzer.py

```diff
@@ -386,7 +386,8 @@ def __init__(self,
                  operations_interval=1,
                  max_partitions=10,
                  min_replication=1,
-                 max_replication=3):
+                 max_replication=3,
+                 allowed_operations=[o for o in RedpandaAdminOperation]):
```

Comment:
I would avoid using a list as the default for an argument as it's mutable and can have surprising behaviours. See https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments.

Reply:
good point, will fix